<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicBackgroundSkinDialog</class>
 <widget class="QWidget" name="MusicBackgroundSkinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>皮肤与窗口调整</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>620</width>
     <height>405</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>皮肤与窗口调整</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="MusicClickedLabel" name="mySkin">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>48</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>我的</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MusicClickedLabel" name="netSkin">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>48</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>推荐</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MusicAnimationStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>77</y>
     <width>585</width>
     <height>300</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="MusicBackgroundListWidget" name="themeListWidget"/>
   <widget class="MusicBackgroundRemoteWidget" name="remoteWidget"/>
  </widget>
  <widget class="QPushButton" name="customSkin">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>385</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>自定义皮肤</string>
   </property>
  </widget>
  <widget class="QPushButton" name="paletteButton">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>385</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>调色板</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>620</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <widget class="MusicBackgroundPopWidget" name="listTransparentButton" native="true">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>385</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="listTransparentLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>385</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>列表透明</string>
   </property>
  </widget>
  <widget class="QLabel" name="skinTransparentLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>385</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>皮肤透明</string>
   </property>
  </widget>
  <widget class="MusicBackgroundPopWidget" name="skinTransparentButton" native="true">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>385</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="arrowLabel">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>70</y>
     <width>9</width>
     <height>7</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="resetWindowButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>55</y>
     <width>118</width>
     <height>13</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>mySkin</zorder>
  <zorder>netSkin</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>customSkin</zorder>
  <zorder>paletteButton</zorder>
  <zorder>listTransparentLabel</zorder>
  <zorder>skinTransparentLabel</zorder>
  <zorder>listTransparentButton</zorder>
  <zorder>skinTransparentButton</zorder>
  <zorder>arrowLabel</zorder>
  <zorder>resetWindowButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicBackgroundListWidget</class>
   <extends>QListWidget</extends>
   <header>musicbackgroundlistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicBackgroundRemoteWidget</class>
   <extends>QWidget</extends>
   <header>musicbackgroundremotewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicBackgroundPopWidget</class>
   <extends>QWidget</extends>
   <header>musicbackgroundpopwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicAnimationStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>musicanimationstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicClickedLabel</class>
   <extends>QLabel</extends>
   <header>musicclickedlabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
